cardano-node/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -234,7 +232,7 @@ mkSomeConsensusProtocolCardano NodeByronProtocolConfiguration {
         , triggerHardForkConway =
              case npcTestConwayHardForkAtEpoch of
                 Nothing -> Consensus.TriggerHardForkAtVersion
-                             (maybe 9 fromIntegral npcTestConwayHardForkAtVersion)
+                             (maybe 10 fromIntegral npcTestConwayHardForkAtVersion)
```

Comment:
The HFC uses this major ledger protocol version to determine when to HF to Conway; therefore, this needs to stay at 9 (otherwise, we are incompatible with any existing Conway chain). 
```suggestion
                             (maybe 9 fromIntegral npcTestConwayHardForkAtVersion)
```